modules/api/api-xsjs/src/main/resources/META-INF/dirigible/xsk/web/web.js

```diff
@@ -292,9 +297,9 @@ exports.WebRequest = function (method, path) {
   }
 
   function transformParametersObject(dParametersObject) {
-    var arrayToReturn = [];
+    let arrayToReturn = [];
```

Comment:
Use `const`

Reply:
Values are pushed to arrayToReturn. Cont will not be OK

Reply:
This is not correct. `const` means the variable can't be reassigned, not that you can't modify it. Refer to MDN for additional docs

Reply:
Just read. 
Will switch to const here and in all other places.